### PR TITLE
Update Github Event Collector integration

### PR DESCRIPTION
**Pull request by adi88d**
add description

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Pull request:
Fixed version for the last comments in the PR: https://github.com/demisto/content/pull/18863

## Description
- add modeling rules.
- call `send_events_to_xsiam` also in `github-get-events` command.
- call `send_events_to_xsiam`with empty list when there are no new events.
- add `product` parameter for configurate the product name when creating the events.
- update some descriptions.
- add `GitHubEventCollector_description.md`

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector_description.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Integration with Github using REST API to Gets the audit log for an organization.
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector_description.md:1]:
```suggestion
Integration with Github using REST API to Get the audit log for an organization.
```

File: Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector_description.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+## Configuration Parameters
+
+**Server URL**  
+Endpoit to get the logs, Replace the `${ORGANIZATION}` value with your organization: ``https://api.github.com/orgs/${ORGANIZATION}/audit-log``
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector_description.md:6]:
```suggestion
Endpoint to get the logs, Replace the `${ORGANIZATION}` value with your organization: ``https://api.github.com/orgs/${ORGANIZATION}/audit-log``
```

File: Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector_description.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+**API Key**  
+access toke you created in github, You must be an organization owner, and you must use an access token with the admin:org scope.
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector_description.md:9]:
```suggestion
access token created in GitHub, You must be an organization owner, and you must use an access token with the admin:org scope.
```

File: Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.yml [hunk 4]
```diff
@@ -41,11 +43,16 @@ configuration:
...
+  defaultvalue: 'github-audit'
+  required: true
 - display: Use system proxy settings
   name: proxy
   type: 8
   required: false
-- display: Use Secured Connection
+- display: Trust any certificate (not secure)
   name: verify
   defaultvalue: "true"
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.yml:57]:
```suggestion
```

File: Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.py [hunk 5]
```diff
@@ -184,6 +131,12 @@ def main():
                     raw_response=events,
                 )
                 return_results(command_results)
+
+            while len(events) > 0:
```
Comment (yaakovi) [inline Packs/GitHub/Integrations/GitHubEventCollector/GitHubEventCollector.py:135]: no need for batching, you can send it in one request.
```suggestion
            while events:
```

**Review by yaakovi — APPROVED** [review]

**Review by ShahafBenYakir — APPROVED** [review]
seems ok

**Pull request merged**